compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -3708,8 +3708,8 @@ object Parsers {
 
     /** with Template, with EOL <indent> interpreted */
```

Comment:
should this comment be updated? Is this comment a pseudo grammar? Just from the comment it is not clear which "with" is a keyword and whether "," is a token or an informal alternative.